components/dashboard/package.json

```diff
@@ -9,13 +9,15 @@
     "@gitpod/public-api": "0.1.5",
     "@stripe/react-stripe-js": "^1.7.2",
     "@stripe/stripe-js": "^1.29.0",
+    "@types/react-datepicker": "^4.8.0",
```

Comment:
**issue**: Besides all the rendering and interaction issues of this library, I've heard multiple accessibility issues around it, see [relevant blog post](https://adrianroselli.com/2019/07/maybe-you-dont-need-a-date-picker.html#:~:text=Unnecessary%20detail%20about%20one%20library%E2%80%99s%20date%20picker%E2%80%A6). :x:

Reply:
The decision for `react-datepicker` is easily revertable as we really only use the element. So if at some point in future we believe it's worth investing in an alternative we can easily replace it.

Reply:
DEAL—Let's try to bring the current library to offer a bare minimum good UX. I can try to make some visual changes in a follow-up PR if needed. 🤝